run_ci.sh

```diff
@@ -140,6 +140,10 @@ for PROJ in installer ; do
     mv /home/notstack/$PROJ $HOME/go/src/github.com/$GITHUB_ORGANIZATION
 done
 
+# If you wish the deployment to install a custom MAO then set the following to true
+# Make sure to specify the custom MAO image in 11_register_hosts.sh
```

Comment:
The config shouldn't be specified in 11_register_hosts.sh, it should be provided in the environment, e.g for CI here, and for local testing via `config_$USER.sh`